### ISSUE TITLE
spectrogram (image_-to-wav

### ISSUE THREAD

**Issue by ahmeftah** [fatchord/WaveRNN#247]
Dear Authors and readers

I would appreciate it if you would give me an answer to my question:
Is it possible to convert the spectrogram image (not the array) to wav (reconstruct the wav audio from the spectrogram in image form)?

**Comment by magicse** [comment]
Yes this possible. But it will be scaled to 0..255 range as image. Before use it need again normalize something like this for exaple image * (1/255)  * 13 -11 and you will get range of mel spectrogram -11..2
![melgram_flipped](https://github.com/fatchord/WaveRNN/assets/13585785/053ff7e5-f7ce-499b-bab7-04733d5cf6f5)
